docs/useTabs.md

```diff
@@ -0,0 +1,47 @@
+---
+id: use-tabs
+title: useTabs
+sidebar_label: useTabs
+---
+
+## Lorem
```

Comment:
Does this text block in the right place? I guess this was not supposed to be here.

Reply:
I think we can do better when create the doc website. For while, it's fine.